isis/src/qisis/objs/MosaicSceneWidget/GridGraphicsItem.cpp

```diff
@@ -283,16 +289,20 @@ namespace Isis {
             while (!atMaxLat) {
               double x = 0;
               double y = 0;
-              bool valid = tproj->SetUniversalGround(lat.degrees(), lon.degrees());
+                
+              bool valid;
+              if (tproj->IsPositiveWest()) {
+                valid = tproj->SetGround(lat.degrees(), lon.positiveWest(Angle::Degrees));
+              }
+              else {
+                valid = tproj->SetGround(lat.degrees(), lon.positiveEast(Angle::Degrees));
+              }
 
               if (valid) {
                 x = tproj->XCoord();
                 y = -1 * tproj->YCoord();
 
                 if(havePrevious) {
-                  x = proj->XCoord();
-                  y = -1 * proj->YCoord();
```

Comment:
Why'd this get removed? Just kinda curious what the difference is between the tproj and the proj object

Reply:
I second this question

Reply:
It's duplication of the step above without any changes. It'll overwrite itself with the exact same value.